### ISSUE TITLE
Can't open the website that you offer

### ISSUE THREAD

**Issue by Maxiah** [nmoehrle/mvs-texturing#194]
Is that change the domain name or something?
https://www.gcc.tu-darmstadt.de/home/proj/texrecon/
If there is a new website, please tell me.

**Comment by ShiqiMa** [comment]
Hi, did you find out the new website?
Without the guide, I don't know what to do next.

**Comment by Maxiah** [comment]
> Hi, did you find out the new website? Without the guide, I don't know what to do next.
> 
> 嗨，你找到新网站了吗? 没有指南，我不知道下一步该怎么办。

Nope, I found their school's website: https://www.tu-darmstadt.de/ but I can't tell this is or not, because I search the key word 'mve' but find nothing.

**Comment by DeadlyMagikarps** [comment]
> Hi, did you find out the new website? Without the guide, I don't know what to do next.

It's based off mve here: https://github.com/simonfuhrmann/mve/wiki/MVE-Users-Guide

But otherwise, I have no idea what it used to be. I wish I snapshotted the website before it went down.

**Comment by citystrawman** [comment]
Same problem, dont know how to get the tutorial.

**Comment by sjobeek** [comment]
FYI an archived version of the project page can be found at the internet archive here:

https://web.archive.org/web/20211021152444/https://www.gcc.tu-darmstadt.de/home/proj/texrecon/
